agent/lib/kontena/network_adapters/ipam_client.rb

```diff
@@ -73,7 +73,12 @@ def release_address(network, address)
       debug "response: #{response.status}/#{response.body}"
       JSON.parse(response.body)
     rescue Excon::Errors::HTTPStatusError => error
-      handle_error_response(error)
+      if error.response.status == 409
+        # IPAM return 409 in case the address still responds to ping, case zombies
+        warn error.response.body
```

Comment:
I would think that it would be preferable to just drop any release errors. AFAIK letting the current weave actor crash is not going to have any advantages, it doesn't really affect any recovery.